pkg/defaults/defaults_windows.go

```diff
@@ -0,0 +1,55 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package defaults
+
+import (
+	"fmt"
+)
+
+const AppArmorProfileName = ""
+
+func DataRoot() string {
+	return "C:\\ProgramData\\containerd\\root"
```

Comment:
Should read `%ALLUSERSPROFILE`?

Reply:
Specifically,
```
filepath.Join(os.Getenv("ProgramData"), "containerd", "root")
```
is how [`containerd.defaults.DefaultRootDir` is defined](https://github.com/containerd/containerd/blob/2486eb8ff8cf36b189108661d8e7f809fc288b60/defaults/defaults_windows.go#L27-L29).

Reply:
The "data root" of nerdctl has nothing to do with containerd's data root.
So this should be like `filepath.Join(os.Getenv("ProgramData"), "nerdctl")`